Libraries/Components/Button.windows.js

```diff
@@ -116,6 +143,32 @@ class Button extends React.Component {
     );
     const formattedTitle = Platform.OS === 'android' ? title.toUpperCase() : title;
     const Touchable = Platform.OS === 'android' ? TouchableNativeFeedback : TouchableOpacity;
+    var content =
+        <View style={buttonStyles}>
+          <Text style={textStyles} disabled={disabled}>{formattedTitle}</Text>
+        </View>;
+
+    if (Platform.OS === "windows") {
+      var tabIndex = this.props.tabIndex || 0;
+      var windowsTabFocusable = !disabled && tabIndex >= 0;
+      content =
+        <FocusableWindows
+          ref="focusable"
```

Comment:
>ref="focusable" [](start = 10, length = 15)

React has deprecated string-based refs in favor of callback-based refs.

Reply:
Oops.

Reply:
Oops, fixed.

---
In reply to: [156557625](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r156557625) [](ancestors = 156557625)

Reply:
Avoid inline functions in JSX when possible. To read about why it's bad for refs in particular, read here: [https://reactjs.org/docs/refs-and-the-dom.html#caveats](https://reactjs.org/docs/refs-and-the-dom.html#caveats)

---
In reply to: [156843044](https://github.com/Microsoft/react-native-windows/pull/1546#discussion_r156843044) [](ancestors = 156843044,156557625)